spec/models/tweet_spec.rb

```diff
@@ -1,4 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe Tweet, type: :model do
+
+  describe '.default scope' do
+    it 'orders by most recently posted' do
+      tweet_one = FactoryGirl.create(:tweet)
+      expect(Tweet.first).to eq tweet_one
```

Comment:
you might want to create a few tweets and then test that the first one created is actually first.  this test will always pass because you are only creating one tweet (it looks like).